microsoft-edge/extensions-chromium/developer-guide/manifest-v3.md

```diff
@@ -32,6 +32,8 @@ To help you plan, consider the following schedule for Microsoft Partner Center a
 
 This timeline is subject to change. This article will be updated to share exact details as the milestones draw nearer.
 
+We acknowledge that Chrome has revised the timelines for Manifest V2 sunset. We are in the process of updating this document. Watch this space for more updates.
```

Comment:
```suggestion

#### Revised timeline

This timeline is subject to change. This article will be updated as the milestones draw nearer.  Chrome has revised the timeline for Manifest V2 sunset, and we are about to update this article to reflect that change.
```
This Suggestion reduces wordcount to focus the communication, and creates a section by adding a small heading.  I plan to create a separate PR containing a general structural edit of this article, to enable this PR to be merged now.
@Reezaali  @sonakshimore

Reply:
Committing based on discussion with Michael.